src/components/v2v3/V2V3Project/ProjectPageMobile.tsx

```diff
@@ -0,0 +1,36 @@
+import { NftRewardsSection } from 'components/NftRewards/NftRewardsSection'
+import { PayProjectForm } from 'components/Project/PayProjectForm'
+import { V2V3ProjectContext } from 'contexts/v2v3/Project/V2V3ProjectContext'
+import { useValidatePrimaryEthTerminal } from 'hooks/v2v3/ValidatePrimaryEthTerminal'
+import { useContext } from 'react'
+import ProjectActivity from './ProjectActivity'
+import { ProjectPageTabs } from './ProjectPageTabs/ProjectPageTabs'
+
+export function ProjectPageMobile({
+  hasNftRewards,
+}: {
+  hasNftRewards: boolean
+}) {
+  const { fundingCycle } = useContext(V2V3ProjectContext)
+  const hasCurrentFundingCycle = fundingCycle?.number.gt(0)
+  const isPrimaryETHTerminalValid = useValidatePrimaryEthTerminal()
+
+  const payFormDisabled = !hasCurrentFundingCycle || !isPrimaryETHTerminalValid
+
+  return (
+    <>
+      <section>
+        <PayProjectForm disabled={payFormDisabled} />
```

Comment:
**note (non0-blocking)** tech debt, would be reasonable to have a `V2V3PayProjectForm` so that the pay form disabled logic isn't duplicated.